CMakeLists.txt

```diff
@@ -291,23 +291,27 @@ if(PHYSFS_BUILD_DOCS)
 endif()
 
 if(UNIX)
-    set(PHYSFS_TARBALL "${CMAKE_CURRENT_SOURCE_DIR}/../physfs-${PHYSFS_VERSION}.tar.gz")
-
-    set(PHYSFS_TARGETNAME_DIST "dist" CACHE STRING "Name of 'dist' build target")
-    add_custom_target(
-        ${PHYSFS_TARGETNAME_DIST}
-        git archive --prefix="physfs-${PHYSFS_VERSION}/" --output="${PHYSFS_TARBALL}" HEAD
-        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-        COMMENT "Building source tarball '${PHYSFS_TARBALL}'..."
-    )
+    option(PHYSFS_DISABLE_TARBALL "Disable installing PhysFS tar.gz" OFF)
```

Comment:
SDL conditionally disables installation, and distribution-related options, [when it is built as a subdirectory](https://github.com/libsdl-org/SDL/blob/dfd8073a8fb3496b97929cbf445aa54a11092ba4/CMakeLists.txt#L266).
The detection happens [here](https://github.com/libsdl-org/SDL/blob/dfd8073a8fb3496b97929cbf445aa54a11092ba4/CMakeLists.txt#L13-L17).

Also, I would drop the `if(UNIX)`. If it's up to me, Windows users can also create a distribution. But icculus isn't :)